frontendUiRootImpl/src/main/java/com/virtuslab/gitmachete/frontend/ui/impl/GitMacheteErrorHandler.java

```diff
@@ -0,0 +1,31 @@
+package com.virtuslab.gitmachete.frontend.ui.impl;
+
+import java.awt.Component;
+
+import com.intellij.ide.BrowserUtil;
+import com.intellij.openapi.application.ModalityState;
+import com.intellij.openapi.diagnostic.ErrorReportSubmitter;
+import com.intellij.openapi.diagnostic.IdeaLoggingEvent;
+import com.intellij.openapi.diagnostic.SubmittedReportInfo;
+import com.intellij.ui.GuiUtils;
+import com.intellij.util.Consumer;
+import org.checkerframework.checker.nullness.qual.Nullable;
+
+public class GitMacheteErrorHandler extends ErrorReportSubmitter {
+  @Override
+  public String getReportActionText() {
+    return "Open Issue On GitHub";
+  }
+
+  @Override
+  public boolean submit(
+      IdeaLoggingEvent[] events,
+      @Nullable String additionalInfo,
+      Component parentComponent,
+      Consumer<SubmittedReportInfo> consumer) {
+    GuiUtils.invokeLaterIfNeeded(() -> BrowserUtil.browse(
+        "https://github.com/VirtusLab/git-machete-intellij-plugin/issues/new?assignees=&labels=bug&template=bug_report.md"),
+        ModalityState.NON_MODAL);
+    return super.submit(events, additionalInfo, parentComponent, consumer);
```

Comment:
Hmm so what's gonna happen in this `super` call? Isn't it enough to just stick to `BrowserUtil.browse`?

Reply:
right, no need 

returning `true` 

![image](https://user-images.githubusercontent.com/19799111/93474071-2afc2680-f8f7-11ea-8df9-e7a754a2818d.png)